theming.js

```diff
@@ -360,7 +360,8 @@ const Transparency = new Lang.Class({
 
         // Window signals
         global.get_window_actors().forEach(function(win) {
-            this._addWindowSignals(win);
+            if (win.get_meta_window().get_wm_class() !== 'Gnome-shell')
```

Comment:
This works upstream because they never disconnect the signal? How do you reproduce the bug?

Reply:
Exactly, it's a bit annoying, but sometimes I can trigger this. I tried to pinpoint the root cause of this, but I was unable to. There are some actors that don't get properly connected, and then fail disconnection. As you guessed, the `panel` signals are never disconnected and hence no warning appears.

Reply:
Ok: we have the same problem in the intellihide class https://github.com/micheleg/dash-to-dock/blob/master/intellihide.js#L107-L109

Reply:
I don't use intellihide, so perhaps this is why I didn'd encounter the error before. Did you see it anywhere else? Like so:

```
gnome-shell[1313]: /build/glib2.0-B1uXKV/glib2.0-2.50.3/./gobject/gsignal.c:2641: instance '0x55fc2d1d47f0' has no handler with id '52060'
```

where `0x55fc2d1d47f0` is a window instance (I did check this in the past).